### PR TITLE
Make Hydra's web ACLs look like Fedora web ACLs

### PR DESCRIPTION
**Pull request by jcoyne**

### CODE REVIEW

**Comment by tpendragon** [comment]
Testing in console looks good. Not sure how to get Fedora to tell me it's using these or not, and I'm not sure Fedora supports the URIs that this PR puts in as the agent string.

Maybe we should add a type statement to identify an agent as a user or group?

**Comment by tpendragon** [comment]
Plum's tests pass with this.

File: hydra-access-controls/app/models/hydra/access_control.rb [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+    def permissions=(records)
+      relationship.replace(records)
+    end
+
+    def permissions_attributes=(attribute_list)
+      raise ArgumentError unless attribute_list.is_a? Array
+      attribute_list.each do |attributes|
+        if attributes.key?(:id)
+          obj = relationship.find(attributes[:id])
+          if attributes.key?('_destroy') && %w{1 true}.include?(attributes['_destroy'])
```
Comment (cbeer) [inline hydra-access-controls/app/models/hydra/access_control.rb]: Can we use  https://github.com/projecthydra/active_fedora/blob/a58ee090a1564f6098271905b1968ca46565ef36/lib/active_fedora/type/boolean.rb here?

File: hydra-access-controls/spec/unit/permissions_spec.rb [hunk 2]
```diff
@@ -101,6 +109,7 @@ class Foo < ActiveFedora::Base
           expect(subject.permissions.to_a).to all(be_a(Hydra::AccessControls::Permission))
           expect(subject.permissions[0].to_hash).to eq(type: "person", access: "edit", name: "jcoyne")
           expect(subject.permissions[1].to_hash).to eq(type: "person", access: "edit", name: "user1")
+          # expect(subject.permissions[2].to_hash).to eq(type: "person", access: "edit", name: "user1")
```
Comment (cbeer) [inline hydra-access-controls/spec/unit/permissions_spec.rb]: Can we remove this commented line?

**Pull request merged**